src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx

```diff
@@ -229,14 +226,11 @@ export const StopLossLevelUp = () => {
                                     translate_text="<0> 3351.46</0>"
                                     components={[<FormulaGreen key={0} />]}
                                 />
-                                <PointerContainerMobile mw="40px">
+                                <PointerContainerMobile mw="0" ml="-20px">
                                     <PointerDotMobile />
-                                    <PointerStickMobile />
-                                    <PointerTextMobile>
-                                        <Localize translate_text="Stop loss" />
-                                    </PointerTextMobile>
+                                    <PointerStickMobile height="18px" />
                                     <PointerTextMobile>
-                                        <Localize translate_text="level" />
+                                        <Localize translate_text="Stop loss level" />
```

Comment:
Do we need translation for this?

Reply:
@prince-deriv I think Yes, that's why they wanted to convert from image to code

Reply:
@meenakshi-deriv do you know if they need the translation right away or we can have it later?

Reply:
@habib-deriv Can you please advice